<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceClass</class>
 <widget class="QMainWindow" name="InterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InterfaceTut</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QTabWidget" name="Onglets">
        <property name="minimumSize">
         <size>
          <width>540</width>
          <height>671</height>
         </size>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Cours">
         <attribute name="title">
          <string>Cahier de Cours</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <spacer name="horizontalSpacer_21">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_22">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="5">
           <spacer name="horizontalSpacer_23">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="6">
           <spacer name="horizontalSpacer_24">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_25">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_26">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="7">
           <spacer name="horizontalSpacer_27">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="9">
           <widget class="QPushButton" name="PleinEcranCours">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Resources/Fullscreen.gif</normaloff>Resources/Fullscreen.gif</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <spacer name="horizontalSpacer_28">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_29">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="7">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Exo">
         <attribute name="title">
          <string>Cahier d'Exercices</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="1">
           <widget class="QPushButton" name="PleinEcranExo">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Resources/Fullscreen.gif</normaloff>Resources/Fullscreen.gif</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_30">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Eval">
         <attribute name="title">
          <string>Cahier d'Ã‰valuation</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="1" column="1">
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="PleinEcranEval">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Resources/Fullscreen.gif</normaloff>Resources/Fullscreen.gif</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_31">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QWidget" name="Geom" native="true">
        <property name="minimumSize">
         <size>
          <width>549</width>
          <height>689</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="9">
          <widget class="QPushButton" name="GeomPleinEcran">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Resources/Fullscreen.gif</normaloff>Resources/Fullscreen.gif</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_32">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_33">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_34">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_35">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_36">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="5">
          <spacer name="horizontalSpacer_37">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="6">
          <spacer name="horizontalSpacer_38">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="7">
          <spacer name="horizontalSpacer_39">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="8">
          <spacer name="horizontalSpacer_40">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="7">
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouvel_Utilisateur"/>
    <addaction name="actionChanger_d_Utilisateur"/>
    <addaction name="actionSupprimer_un_Utilisateur"/>
    <addaction name="separator"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionSauvegarder_Sous"/>
    <addaction name="separator"/>
    <addaction name="actionMise_en_Page"/>
    <addaction name="actionApercu_avant_impression"/>
    <addaction name="actionImprimer"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionRefaire"/>
    <addaction name="separator"/>
    <addaction name="actionCopier"/>
    <addaction name="actionColler"/>
    <addaction name="actionCouper"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionGrille"/>
    <addaction name="actionContraste"/>
    <addaction name="actionR_glages_magn_tisme"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionRaccourcis_Clavier"/>
    <addaction name="actionPrise_en_main_du_logiciel"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouvel_Utilisateur">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ajouterUti.png</normaloff>Resources/ajouterUti.png</iconset>
   </property>
   <property name="text">
    <string>Nouvel Utilisateur</string>
   </property>
  </action>
  <action name="actionChanger_d_Utilisateur">
   <property name="icon">
    <iconset>
     <normaloff>Resources/modif_uti.png</normaloff>Resources/modif_uti.png</iconset>
   </property>
   <property name="text">
    <string>Changer d'Utilisateur</string>
   </property>
  </action>
  <action name="actionSupprimer_un_Utilisateur">
   <property name="icon">
    <iconset>
     <normaloff>Resources/supprUti.png</normaloff>Resources/supprUti.png</iconset>
   </property>
   <property name="text">
    <string>Supprimer un Utilisateur</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="icon">
    <iconset>
     <normaloff>Resources/sauver.png</normaloff>Resources/sauver.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionSauvegarder_Sous">
   <property name="icon">
    <iconset>
     <normaloff>Resources/sauver_sous.png</normaloff>Resources/sauver_sous.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder Sous</string>
   </property>
  </action>
  <action name="actionMise_en_Page">
   <property name="icon">
    <iconset>
     <normaloff>Resources/miseEnPage.png</normaloff>Resources/miseEnPage.png</iconset>
   </property>
   <property name="text">
    <string>Mise en Page</string>
   </property>
  </action>
  <action name="actionApercu_avant_impression">
   <property name="icon">
    <iconset>
     <normaloff>Resources/apercuImpression.png</normaloff>Resources/apercuImpression.png</iconset>
   </property>
   <property name="text">
    <string>Apercu avant impression</string>
   </property>
  </action>
  <action name="actionImprimer">
   <property name="icon">
    <iconset>
     <normaloff>Resources/imprimer.png</normaloff>Resources/imprimer.png</iconset>
   </property>
   <property name="text">
    <string>Imprimer</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset>
     <normaloff>Resources/quitter.png</normaloff>Resources/quitter.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="icon">
    <iconset>
     <normaloff>Resources/annuler.png</normaloff>Resources/annuler.png</iconset>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </action>
  <action name="actionRefaire">
   <property name="icon">
    <iconset>
     <normaloff>Resources/refaire.png</normaloff>Resources/refaire.png</iconset>
   </property>
   <property name="text">
    <string>Refaire</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="icon">
    <iconset>
     <normaloff>Resources/copier.png</normaloff>Resources/copier.png</iconset>
   </property>
   <property name="text">
    <string>Copier</string>
   </property>
  </action>
  <action name="actionColler">
   <property name="icon">
    <iconset>
     <normaloff>Resources/coller.png</normaloff>Resources/coller.png</iconset>
   </property>
   <property name="text">
    <string>Coller</string>
   </property>
  </action>
  <action name="actionCouper">
   <property name="icon">
    <iconset>
     <normaloff>Resources/couper.png</normaloff>Resources/couper.png</iconset>
   </property>
   <property name="text">
    <string>Couper</string>
   </property>
  </action>
  <action name="actionRaccourcis_Clavier">
   <property name="icon">
    <iconset>
     <normaloff>Resources/raccourcisClavier.png</normaloff>Resources/raccourcisClavier.png</iconset>
   </property>
   <property name="text">
    <string>Raccourcis Clavier</string>
   </property>
  </action>
  <action name="actionPrise_en_main_du_logiciel">
   <property name="icon">
    <iconset>
     <normaloff>Resources/aide.png</normaloff>Resources/aide.png</iconset>
   </property>
   <property name="text">
    <string>Prise en main du logiciel</string>
   </property>
  </action>
  <action name="actionGrille">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grille</string>
   </property>
  </action>
  <action name="actionContraste">
   <property name="text">
    <string>Contraste</string>
   </property>
  </action>
  <action name="actionR_glages_magn_tisme">
   <property name="text">
    <string>RÃ©glages magnÃ©tisme</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="interfacetut.qrc"/>
 </resources>
 <connections/>
</ui>
